src/macaron/malware_analyzer/pypi_heuristics/heuristics.py

```diff
@@ -6,51 +6,43 @@
 from enum import Enum
 
 
-class Heuristics(Enum):
-    """Seven heuristics for detecting suspicious pypi package.
-
-    Attributes
-    ----------
-    EMPTY_PROJECT_LINK : str
-        Indicates that the package does not contain any project links (such as documentation or Git repository pages).
-    UNREACHABLE_PROJECT_LINKS : str
-        Indicates that the package contains project links, but all of them are unreachable.
-    ONE_RELEASE : str
-        Indicates that the package contains only one release.
-    HIGH_RELEASE_FREQUENCY : str
-        The package has a high release frequency. The average release time (calculated as the sum of all release gaps
-        divided by the number of gaps) is below a set threshold, which defaults to 2 days.
-    UNCHANGED_RELEASE : str
-        Indicates that all releases contain the same content (with identical digests).
-    CLOSER_RELEASE_JOIN_DATE : str
-        Refers to the gap between the date the maintainer registered their account and the date of the latest release.
-    SUSPICIOUS_SETUP : str
-        Indicates that the setup.py file contains suspicious imports, such as base64 and requests.
-    """
+class Heuristics(str, Enum):
+    """Seven heuristics for detecting suspicious pypi package."""
 
+    #: Indicates that the package does not contain any project links (such as documentation or Git repository pages).
     EMPTY_PROJECT_LINK = "empty_project_link"
+
+    #: Indicates that the package contains project links, but all of them are unreachable.
     UNREACHABLE_PROJECT_LINKS = "unreachable_project_links"
+
+    #: Indicates that the package contains only one release.
     ONE_RELEASE = "one_release"
+
+    #: The package has a high release frequency. The average release time (calculated as the sum of all release gaps
+    #: divided by the number of gaps) is below a set threshold, which defaults to 2 days.
```

Comment:
nit: I think we shouldn't mention the default value in this Enum description but instead put it within the heuristic implementation itself or `defaults.ini`.

Reply:
See commit [8f76fc0](https://github.com/oracle/macaron/pull/797/commits/8f76fc04e6464408bdc28debf99adc079a567281)